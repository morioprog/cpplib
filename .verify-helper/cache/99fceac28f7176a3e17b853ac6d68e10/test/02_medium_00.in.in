30
4 1 2 3 4
0
4 5 6 7 8
1 23
0
0
3 9 10 11
0
0
0
0
2 12 13
2 14 15
0
0
3 16 17 18
0
0
1 19
1 20
2 21 22
0
3 27 28 29
2 24 25
1 26
0
0
0
0
0
12
0 19 1
0 17 2
0 13 3
0 9 4
1 16
1 6
1 26
1 9
1 29
0 28 5
1 29
1 28
