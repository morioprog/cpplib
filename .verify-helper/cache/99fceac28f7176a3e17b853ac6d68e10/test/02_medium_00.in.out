32
20
0
24
34
79
84
